.env.sample

```diff
@@ -33,9 +33,9 @@ L1_ENDPOINT_WS=
 L1_BEACON_HTTP=
 
 # REQUIRED FOR PROVER
-# If you are running your Taiko node on a separate server from your sgx prover, please provide the server hosting the taiko node's public IP address here (no slash at the end).
-# Otherwise, if you are running both on one machine, set it to http://l2_execution_engine.
-TAIKO_NODE_IP=
+# In put Path to a JWT secret for the Raiko service
+
+RAIKO_JWT=
```

Comment:
only for advanced users like you :)

Reply:
Haha.. I think team is going to add authentication for Raiko since it will able to run on separated server.. And now Taiko calling Raiko public without authentication.
Feel free close this PR.
The most important part https://github.com/taikoxyz/simple-taiko-node/pull/273/files#diff-bccdbf394f67f6e0186bab7b4a449ea9b88ef9fe5dc7a16d0f7058286ec2e96aL26
already in PR [#26](https://github.com/taikoxyz/simple-taiko-node/pull/276/files)
I just have question main branch will able to work with A7 testnet since boos_node and chain ID has changed ?

Reply:
Thanks @RogerLamTd .. will close this PR